### PR TITLE
transitioned to responsive file, progress on displaying more politici…

### PR DESCRIPTION
**Pull request by Hippopattymus**
…an content

### CODE REVIEW

File: assets/javascript/code.js [hunk 1]
```diff
@@ -26,26 +26,30 @@ function displayInfo() {
             }
 
             if (pol == (Fname + " " + Lname)) {
-                console.log(pol == (Fname + " " + Lname));
-                var polDiv = $("<div class = 'card'>");
+                
+                var polDiv = $("<div class = 'rounded col-lg-6 col-xs-6' width= 250px>");
```
Comment (jmckenney) [inline assets/javascript/code.js]: generally no spaces around the `=`. Not a big deal.

Comment (jmckenney) [inline assets/javascript/code.js]:
```suggestion
                var polDiv = $("<div class='rounded col-lg-6 col-xs-6' width='250px'>");
```

File: assets/javascript/code.js [hunk 2]
```diff
@@ -26,26 +26,30 @@ function displayInfo() {
...
                 } else {
                     polDiv.addClass("bg-warning");
                 }
-
-                var p = $("<p class = 'pt-2'>").text(response.officials[i].name);
+                
+                var p = $("<p class = 'pt-2 candidateText'>").text(response.officials[i].name);
                 p.append("<hr>" + response.officials[i].party);
-                p.addClass("float-left");
+                p.append("<hr>" + response.officials[i].phones[0]);
```
Comment (jmckenney) [inline assets/javascript/code.js:43]: I don't believe hrs should be inside of paragraphs.

File: assets/javascript/code.js [hunk 3]
```diff
@@ -26,26 +26,30 @@ function displayInfo() {
...
+                p.append("<hr>" + response.officials[i].urls[0]);
+                p.append("<hr> Lorem ipsum dolor sit amet, consectetur adipisicing elit Quibusdam ipsam suscipit, quae inventore doloribus ipsa");
+                
 
-                var img = $("<img height = 100px width = 100px>");
+                var img = $("<img class = 'candidatePicture' height= 150px width= 150px>");
```
Comment (jmckenney) [inline assets/javascript/code.js]:
```suggestion
                var img = $("<img class='candidatePicture' height='150px' width='150px'>");
```

File: assets/javascript/code.js [hunk 4]
```diff
@@ -26,26 +26,30 @@ function displayInfo() {
...
                 img.attr("src", response.officials[i].photoUrl);
-                img.addClass("pt-2");
-
-                polDiv.addClass("float-left");
-                polDiv.addClass("col-6");
+                img.addClass("pt-3");
+                //img.addClass("float-left");
```
Comment (jmckenney) [inline assets/javascript/code.js]: Don't forget to clean up comments.

**Review by fxtzhou — APPROVED** [review]
accepted

**Pull request closed without merging**